packages/accordion/components/StatelessAccordion.tsx

```diff
@@ -0,0 +1,42 @@
+import * as React from "react";
+import { Stack } from "../../styleUtils/layout";
+import { Provider as AccordionProvider } from "./AccordionContext";
+import { AccordionBaseProps } from "../types";
+
+interface AccordionProps extends AccordionBaseProps {
+  /**
+   * An array of open accordion panel IDs
+   */
+  expandedItems?: string[];
+}
+
+const Accordion: React.FC<AccordionProps> = ({
+  allowMultipleExpanded,
+  "data-cy": dataCy,
+  children,
+  expandedItems = [],
+  onChange
+}) => {
+  //   React.useEffect(() => {
+  //     if (onChange) {
+  //       onChange(expandedItems);
+  //     }
+  //   }, [expandedItems]);
+  //   console.log("expandedItems", expandedItems);
+  return (
+    <AccordionProvider
+      allowMultipleExpanded={allowMultipleExpanded}
+      controlledExpandedItems={expandedItems}
+      onChange={onChange}
+    >
+      <Stack data-cy={dataCy}>{children}</Stack>
+    </AccordionProvider>
+  );
+};
+
+Accordion.defaultProps = {
+  allowMultipleExpanded: false,
+  "data-cy": "accordion"
+};
+
+export default Accordion;
```

Comment:
Just curious, why did you decide to go with a new component, vs. being able to use the accordion in a controlled and uncontrolled way (like one would use inputs)

Reply:
Because eliminating the prop `initialExpandedItems` would be a breaking change, and I didn't want to add new prop just to allow the Accordion to be used in a controlled way.